docs/pages/server-access/guides/ssh-pam.mdx

```diff
@@ -71,8 +71,26 @@ own PAM service file like `/etc/pam.d/teleport` and specifying it as
 
 ## Set Message of the Day (motd) with Teleport
 
-The file `/etc/motd` is normally displayed by login(1) after a user has logged in
-but before the shell is run. It is generally used for important system-wide announcements.
+A cluster-wide Message of the Day can be set in the `auth_server` configuration.
+
+```yaml
+auth_service:
+    message_of_the_day: "Welcome to the cluster. All activity will be logged."
+```
```

Comment:
It looks like the text and example might not agree - `auth_server` vs `auth_service`

Reply:
Fixed